spec/classes/foreman_plugin_remote_execution_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::remote_execution' do
+  let :facts do
+    on_supported_os['redhat-7-x86_64']
```

Comment:
Yeah, it shouldn't have been accepted, or should have been fixed, which is why we're now asking for them - lack of coverage has caused problems.  Feel free to add one, but it's not really relevant here.